src/api/app/controllers/webui/apidocs_controller.rb

```diff
@@ -20,6 +20,6 @@ def file
       flash[:error] = "File not found: #{params[:filename]}"
       redirect_back(fallback_location: {controller: 'apidocs', action: 'index'})
     end
-    return
+    nil
```

Comment:
Adding a nil there is as ugly as an return, isn't it?

AFAIK here you could just remove the return completely, including the nil. Just be aware of the [double render issue](http://guides.rubyonrails.org/layouts_and_rendering.html#avoiding-double-render-errors) that sometimes require an explicit return.